packages/next/src/server/config.ts

```diff
@@ -1476,3 +1478,22 @@ export function getConfiguredExperimentalFeatures(
   }
   return configuredExperimentalFeatures
 }
+
+function cloneObject(obj: any): any {
+  if (obj === null || typeof obj !== 'object') {
+    return obj
+  }
+
+  if (Array.isArray(obj)) {
+    return obj.map(cloneObject)
+  }
+  const keys = Object.keys(obj)
+  if (keys.length === 0) {
```

Comment:
this is a quick way to check those primitives since we don't have to clone them one by one

Reply:
lets put that as a comment